openedx/core/djangoapps/user_api/serializers.py

```diff
@@ -90,3 +91,14 @@ class CountryTimeZoneSerializer(serializers.Serializer):  # pylint: disable=abst
     """
     time_zone = serializers.CharField()
     description = serializers.CharField()
+
+
+class SoftwareSecurePhotoVerificationSerializer(serializers.ModelSerializer):
+    """
+    Serializer that generates a representation of a user's photo verification status.
+    """
+    expires = serializers.DateTimeField(source='expiration_datetime')
+
+    class Meta(object):
+        fields = ('status', 'expires',)
```

Comment:
Thinking about actual usage, oftentimes we simply want to know if the user is verified. Add a Boolean field, is_verified, that indicates this state. That way clients don't have to parse the expiration time and know about all statuses.